todolist.js

```diff
@@ -0,0 +1,65 @@
+/* Basic Requirements:
+
+Todo list class
+√ set name and owner in constructor
+√ get/set name
+√ get/set owner
+- add Todo item (use the new keyword in a function. what would the name of the instance be?)
+- delete a Todo item (delete keyword? )
+- get list of incomplete todo items ()
+
+Todo item class
+√ set name, description & optional due date in constructor
+- mark item complete/incomplete
+√ get/set title
+√ get/set description
+√ get/set due date
+- check if the item is overdue
+- check if the item is complete */
+
+class ToDoList{
+    constructor(name, owner) {
+        this.name = name;
+        this.owner = owner;
+    }
+    get getName() {
+        return this.name;
+    }
+    set setName(newName) {
+        return this.name = newName;
+    }
+    get getowner() {
+        return this.owner;
+    }
+    set setName(newOwner) {
```

Comment:
I think you meant setOwner here? A test would help catch this kind of typo - this is an example of why we recommend adding tests before writing methods.

```suggestion
    set setOwner(newOwner) {
```